doc/lms.mysql

```diff
@@ -3686,6 +3686,7 @@ URL: %url
 ('userpanel', 'pin_validation', 'true', '', 0),
 ('userpanel', 'show_all_assignments', 'false', '', 0),
 ('userpanel', 'allowed_document_types', '', '', 0),
+('userpanel', 'force_ssl', 'false', '', 0),
 ('directories','userpanel_dir','userpanel','',0);
```

Comment:
nie podoba mi się to zmniejszanie bezpieczeństwa domyślnej instalacji w dobie powszechnej dostępności certyfikatów SSL Let's Encrypt generowanych niewielkim nakładem czasu np. narzędziem `certbot`.

Reply:
Myślę, że lepiej uzupełnić instrukcję w wiki o prawidłową instalację userpanelu.
Bo na instancjach bez domeny nie wygeneruje się certyfikat SSL.

Reply:
A do wystawienia na produkcję tak jak ktoś na liście sugerował temu instalatorowi jeszcze daleko z wielu powodów.
Myślę, że każdy admin zdaje sobie z tego sprawę.

Reply:
Z drugiej strony mogę to zmienić na true - ale wtedy każdorazowo przy próbie logowania dostaniemy błąd. Jesteś zdania, że tak lepiej?